guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md

```diff
@@ -36,15 +36,15 @@ Magento 2 has the following {% glossarytooltip 0bc9c8bc-de1a-4a06-9c99-a89a29c30
 
 To view the status of the cache, enter
 
-```terminal
-bin/magento cache:status
+```bash
+   bin/magento cache:status
 ```
 
 <!-- where `--bootstrap=` is a URL-encoded associative array of Magento [application bootstrap parameters]({{ page.baseurl }}/config-guide/bootstrap/magento-how-to-set.html#config-bootparam-overview) and values. -->
 
 A sample follows:
 
-```terminal
+```bash
```

Comment:
```suggestion
```terminal
```

Reply:
bash is for input, terminal is for output. 

Bash statements are rendered with a command line prompt.